### ISSUE TITLE
Ability to suppress stats display for field

### ISSUE THREAD

**Issue by murphyke** [chop-dbhi/cilantro#345]
The default average/max/min and distinct count reports are not helpful in some circumstances, and it would be nice to be able to suppress those on a per-field basis, i.e. have a 'showStats' option analogous to the existing 'showChart' option.

Two examples:

For a field like base pair position on the chromosome, the average/max/min stats are not useful and really just seem to clutter the display.

```
Position
Average 76.4 million Max 249.2 million Min 54
Between _____ and _____
```

Currently, for lexicon fields that are marked enumerable, the "unique values" report is  confusing because it conflicts with the number of values actually shown in the bar chart.  Maybe this should be a separate issue.

```
Chromosome
Unique values 26
[2        ]
[3     ]
[7               ]
[12       ]
```

BTW, I'm happy to attempt to work on easier issues when I have time, but it would seem most efficient to work on an issue only after it has been approved (and perhaps a preferred approach sketched out) by a core committer.
